scripts/deactivate_iac.py

```diff
@@ -0,0 +1,105 @@
+import argparse
+import logging
+import os
+import sys
+
+import requests
+from structlog import configure, wrap_logger
+from structlog.processors import TimeStamper, JSONRenderer
+from structlog.stdlib import add_log_level, filter_by_level, LoggerFactory
+
+from app import config
```

Comment:
Missing app on PYTHONPATH when running script with:

`pipenv run python scripts/deactivate_iac.py`

```suggestion
sys.path.append(os.path.join(os.path.dirname(__file__),
                             os.path.pardir))  # NB: script needs to know about app so append to PYTHONPATH

from app import config  # NOQA
```